src/remediations/write.integration.js

```diff
@@ -234,6 +234,112 @@ describe('remediations', function () {
+            .send({name})
+            .expect(500);
+        });
+
+        test('500s when remediation is created with name used by existing remediation created by a different user within the same org', async () => {
+            const name = 'duplicate name and different users within the same org for create'
+            const {id, header} = reqId();
+
+            const r1 = await request
+            .post('/v1/remediations')
+            .set(header)
+            .set(auth.testWrite2)
+            .send({name})
+            .expect(201);
+
+            const r2 = await request
+            .post('/v1/remediations')
+            .set(header)
+            .set(auth.testWrite)
+            .send({name})
+            .expect(500);
+        });
+
+        test('200s when remediation is created with name used by existing remediation created by a different user in a different org', async () => {
+            const name = 'duplicate name and different users in different orgs for create'
+            const {id, header} = reqId();
+
+            const r1 = await request
+            .post('/v1/remediations')
+            .set(header)
+            .set(auth.default)
+            .send({name})
+            .expect(201);
+
+            const r2 = await request
+            .post('/v1/remediations')
+            .set(header)
+            .set(auth.testWrite)
+            .send({name})
+            .expect(201);
+        });
+
+        test('400s when remediation name is set to empty string', async () => {
+            const name = '';
+            const {id, header} = reqId();
+
+            const {body} = await request
+            .post('/v1/remediations')
+            .set(header)
+            .set(auth.testWrite)
+            .send({name})
+            .expect(400);
+
+            body.errors.should.eql([{
+                id,
+                status: 400,
+                code: 'pattern.openapi.validation',
+                title: 'should match pattern \"^.*[\\w]+.*$\" (location: body, path: name)'
+            }]);
+        });
+
+        test('400s when remediation name is set to null', async () => {
+            const name = null;
+            const {id, header} = reqId();
+
+            const {body} = await request
+            .post('/v1/remediations')
+            .set(header)
+            .set(auth.testWrite)
+            .send({name})
+            .expect(400);
+
+            body.errors.should.eql([{
+                id,
+                status: 400,
+                code: 'type.openapi.validation',
+                title: 'should be string (location: body, path: name)'
+            }]);
+        });
+
+        test('500s when remediation name is undefined upon creation', async () => {
+            const {id, header} = reqId();
+
+            const {body} = await request
+            .post('/v1/remediations')
+            .set(header)
+            .send({auto_reboot: false})
+            .set(auth.testWrite)
+            .expect(500);
```

Comment:
Again - if this causes "internal server error"'s then we need to fix the code.  This should be a 400 with an appropriate message.